### ISSUE TITLE
Summoned Flux Rift Crashes Game

### ISSUE THREAD

**Issue by Orinai** [Azanor/thaumcraft-beta#1060]
So After encountering my first flux rift in beta16, I decided to try and experiment with them to get a better idea of how they work and how to, say , use the stabilizer and such.

but when I used the spawn item for one, The game immediately crashed, and now throws arror and crashes on worldload:

[crash-2018-06-24_15.23.24-client.txt](https://github.com/Azanor/thaumcraft-beta/files/2131182/crash-2018-06-24_15.23.24-client.txt)

Additional helpful (I hope) information:
- to be 100% clear, this only appears to occur when a rift is spawned in manually, not when it spawns naturally through flux production.
- the world begins to load, then goes to a flat sky-blue screen for a few seconds before crashing.
- If I have another active window open (aka minecraft in the background) so that upon world load the game immediately pauses, everything seems to load fine and be visible until I un-pause the game, causing immediate crash.
- MCedit does not seem to recognize that there is an entity at the location of the rift, meaning I cannot remove it via MCedit or similar methods.
- Enabling forge to remove erroring entities does not seem to get rid of the rift, I presume for the same reason as MCedit not recognizing that the entity exists.

Now, of course, I was stupid enough to try these tests in my main world instead of opening a new one, so I'm hoping there is a way to get rid of the entity and recover my world.

And yes, I am aware there is another issue filed concerning this particular bug (#769) I thought it worth filing a new issue, as the problem is reoccurring now in beta16, and the original was never addressed

**Comment by F9Alejandro** [comment]
It seems more of a render issue from what the crashlog says, also please make a gist instead of uploading a file, it will make it easier to access on people with mobile or slow internet speeds.

**Comment by Orinai** [comment]
Update: after loading the world in the background so the pause screen keeps the game from crashing right away, i was able to exit the world and the entity correctly appears in MCedit so it could be removed. So there is a solution for recovering the world, but not for the crash itself

**Comment by wamc2017** [comment]
As Azanor stated [here](https://github.com/Azanor/thaumcraft-beta/issues/962#issuecomment-395014482) flux rifts aren't meant to be spawned in manually...

**Comment by Orinai** [comment]
there was an item Available for spawning them manually, so I tested it, and it caused a crash. isn't that what the Beta and this github is for?

The spawn item did, actually spawn the rift, so it appeared to be a separate issue from #962. Just trying to help by pointing out potential bugs when I find them.

**Comment by wamc2017** [comment]
Yeah, there is an item currently available, but it is not craftable and as Azanor stated, he should maybe remove it. So even if your problem is different to #962 the solution is the same, don't spawn them in (because they are not working or crashing your game) until the item is removed or fixed. 

I didn't say anything that you've done something wrong, bug reports are ok, I only linked the current "solution"... ;)

**Comment by F9Alejandro** [comment]
Instead of removing the item it could just have the links to the function the item performs disabled (or just have the function call removed/commented out)
